src/main/java/org/jabref/gui/groups/GroupTreeViewModel.java

```diff
@@ -430,6 +439,57 @@ private void openAiChat(StringProperty name, ObservableList<ChatMessage> chatHis
         }
     }
 
+    public void generateEmbeddings(GroupNodeViewModel groupNode) {
+        if (currentDatabase.isEmpty()) {
+            dialogService.notify(Localization.lang("Unable to generate embeddings. No library is selected."));
+            return;
+        }
```

Comment:
Is this branch ever it in th execution? Maybe, it can be replaced by a `LOGGER.warn` statement?

Reply:
See https://github.com/JabRef/jabref/pull/11832#discussion_r1785753759

Reply:
porposal for now. Re-use `"Unable to chat"` and add a LOGGER.warn with the full message. If user enounters this situation, we will see in the log.